.gitignore

```diff
@@ -1,4 +1,5 @@
-.DS_Store
-dist
-.cache
 node_modules
+/build
+/*.log
+.*cache
+package-lock.json
```

Comment:
The lock file should always me committed alongside the rest of the code.

Reply:
Since we're using yarn, the mere presence of a package-lock is a smell and git can make sure it is put into evidence.

Reply:
the idea is reducing friction on contribution for people not using yarn.
as long as they don't install any new package, it should be safe.

that, coupled with netlify, as a CI/CD, solves much of the dependency version mismatch issue.

Reply:
as it doesn't interfere with my tools, i have no trouble in removing it the `.gitignore`. 

(:

Reply:
I would advocate for removing it, sure.
We can debate the merits of it in the next co-de-sign haha.

Reply:
```suggestion
- package-lock.json
```

Reply:
![screen shot 2018-10-18 at 11 47 07 am](https://user-images.githubusercontent.com/8649362/47163058-98e2e200-d2cb-11e8-8204-1cc398a9644a.png)
github suggestions 👌